staff/templates/staff_base.html

```diff
@@ -1,221 +1,42 @@
-            '<h4 class="slot-description"></h4><h4 class="slot-description"><img src="https://github.com/Ernesto2004/MB_Portal_Design/blob/master/heart-icon.png?raw=true" width="30" height="30" style="opacity: 0.75">\{\{likes\}\} ' +
-            '<img src="https://github.com/Ernesto2004/MB_Portal_Design/blob/master/comment-icon.png?raw=true" style="opacity: 0.75" width="30" height="30">\{\{model.comments.count\}\}' +
-            '</h4><hr style="width: 90%; height:2px;border:none;color:gray;background-color:gray; opacity: 0.3" />'
-            // CHANGE TEMPLATE IF YOU WANT TO
-      });
-      galleryFeed.run();
-    </script>
-    <script type="text/javascript">
-        jQuery(function($) {
-          var $bodyEl = $('body'),
-              $sidedrawerEl = $('#sidedrawer');
-
-
-          function showSidedrawer() {
-            // show overlay
-            var options = {
-              onclose: function() {
-                $sidedrawerEl
-                  .removeClass('active')
-                  .appendTo(document.body);
-              }
-            };
-
-            var $overlayEl = $(mui.overlay('on', options));
-
-
-            $sidedrawerEl.appendTo($overlayEl);
-            setTimeout(function() {
-              $sidedrawerEl.addClass('active');
-            }, 20);
-          }
-
-
-          function hideSidedrawer() {
-            $bodyEl.toggleClass('hide-sidedrawer');
-          }
-
-
-          $('.js-show-sidedrawer').on('click', showSidedrawer);
-          $('.js-hide-sidedrawer').on('click', hideSidedrawer);
-        });
-    </script>
-    <script>
-    $(document).ready(function(){
-      $(".intro").click(function(){
-        if ($(this).hasClass("intro")){
-            $(this).removeClass("intro").addClass("deintro");
-        }else if($(this).hasClass("deintro")){
-            $(this).removeClass("deintro").addClass("intro");
-        }
-      });
-    });
-
-    $(document).ready(function(){
-      var $rightpanel = $('#mui-panel');
-      $(".instafeedtrigger").click(function() {
-        if ($rightpanel.hasClass('rightpanelbody-hidden')) {
-            $rightpanel.removeClass('rightpanelbody-hidden').addClass('rightpanelbody-visible');
-        } else if ($rightpanel.hasClass('rightpanelbody-visible')){
-            $rightpanel.removeClass('rightpanelbody-visible').addClass('rightpanelbody-hidden');
-        };
-      });
-    });
-    </script>
-    <script>
-      window.onload = function(){
-          document.getElementById('close').onclick = function(){
-              this.parentNode.parentNode.parentNode.parentNode
-              .removeChild(this.parentNode.parentNode.parentNode);
-              return false;
-          };
-      };
-    </script>
+    <!-- NEEDS TO BE MOVED INTO THE STATIC FOLDER
+    BUT FOR SOME REASON 'manipulation_js.html' IS NOT WORKING IN THERE -->
+    {% include 'manipulation_js.html' %}
+    <!-- -->
```

Comment:
You can't really do anything useful with html files in the static folder, you'd want to put .js files in there and script include them. It looks like there's some usage of template tags in there too, which makes it not static. You'd want to leave that as a template or the static code read the settings out of the document, e.g. using [json_script](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#json-script)

Reply:
html files moved to html folder in the project root directory